database/redis/selfstate.go

```diff
@@ -68,7 +69,25 @@ func (connector *DbConnector) SetNotifierState(health string) error {
 	return c.Send("SET", selfStateNotifierHealth, health)
 }
 
+func (connector *DbConnector) GetNotifierMessage() (string, error) {
+	c := connector.pool.Get()
+	defer c.Close()
+	message, err := redis.String(c.Do("GET", selfStateNotifierMessage))
+	if err == redis.ErrNil {
+		message = dto.DefaultMessage
+	}
+	return message, err
+}
+
+func (connector *DbConnector) SetNotifierMessage(message string) error {
```

Comment:
I think that notifier message should be set in a `SetNotifierState`. It will prevent a cases when notifier state is changed but notifier message is not. In this cases notifier state and message will be in inconsisten state and will increase confusion on incident investigation. Especially I can see that in this PR notification message is not changed in self state checker.

Reply:
Yeah that's true, I'm not sure how to go about that though. Can we modify the arguments for `SetNotifierState` in the `Database` interface so that it takes two strings (`state` and `message`)?

Reply:
Yes, this was my point

Reply:
Even more I want to see notifier state as a tuple of two values: boolean state and string message. And I want to see that this two values are changed atomically in one place.